HHS/Head-Start-TTADP
# Submit a report for approval

**Description of change**

A report can now be submitted for approval. Every page must be "completed" before a report can be submitted. A single manager must be assigned as the approving manager.

**How to test**

1) Pull down and run migrations
2) Fill out a complete report
3) Submit the report, note the review page in navigation is marked "submitted"

Or test it out on [dev](https://tta-smarthub-dev.app.cloud.gov/). If you don't see any approving managers available to select make sure your user has READ_WRITE_REPORTS permissions on region 1.

**Issue(s)**
* closes https://github.com/HHS/Head-Start-TTADP/issues/135

**Checklist**

    - API methods
    - Boundary and Data Flow Diagrams
    - [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions with the [Nygard template](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
    - OSCAL templates completed when security controls are implemented or modified
